.github/workflows/pytest.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           python -m pip install coverage coveralls
       - name: Install package
         run: |
-          pip install ".[dev]"
+          pip install ".[dev,apm,em,mpes,xps,stm,xrd]"
```

Comment:
We don't need the plugins here as we are just testing basic pynx functionalities.

Reply:
That came from the previous attempt. I will remove it.

Reply:
removed!